otherlibs/bigarray/Makefile

```diff
@@ -14,22 +14,11 @@
 #**************************************************************************
 
 LIBNAME=bigarray
-EXTRACFLAGS=-I../$(UNIXLIB) -DIN_OCAML_BIGARRAY
-EXTRACAMLFLAGS=-I ../$(UNIXLIB)
-COBJS=mmap_ba.$(O) mmap.$(O)
 CAMLOBJS=bigarray.cmo
+COBJS=empty.$(O)
```

Comment:
It seems that we can't have libraries in otherlibs without C stubs (the construction of bigarray.cmxs fails otherwise). That's why I added this.

Reply:
Invoking C stubs machinery for empty stubs "offends" me (I've jumped through hoops to avoid a similar situation in opam too!)

Reply:
Thanks for the patch!